tox.ini

```
@@ -114,6 +115,30 @@ commands =
     rm -rf {envdir}/.pytest_cache
     make regen
 
+[testenv:plugins]
+changedir = testing/plugins_integration
+deps =
+    django
+    pytest-asyncio
+    pytest-cov
+    pytest-django
+    pytest-html
+    pytest-mock
+    pytest-sugar
+    pytest-trio
+    anyio[curio,trio]
+setenv =
+    PYTHONPATH=.
+commands =
```

```suggestion
commands =
    [ $(pip list -o | wc -l) -eq 2 ]
```

I think we want something like this to make sure there's no version constraints holding us back from the latest releases

/cc @McSinyx

---

As I mentioned [when I suggested this on IRC](http://kafka.dcpython.org/day/pypa/2020-09-13#15.06.54.McSinyx[m]), it's quite of a hack.  I'm not sure how stable pip's output will be but I think at least adding `--quiet` would make it safer.

---

Just run `pip check` and it will fail if any conflicts are detected.